LDAR_Sim/src/programs/method.py

```diff
@@ -428,39 +434,47 @@ def get_daylight_hours(self, daylight, max_hours: int, curr_date) -> int:
             work_hours = max_hours
         return work_hours
 
+    def get_weather_val(self, weather, lat, long, timerange, type) -> float:
```

Comment:
Instead of passing in the whole weather object here, why not just pass in an array of weather values for the timerange

Reply:
For this iteration, splitting out the weather values to grab the relevant weather range is unnecessary extra computation. 
Later down the line when the hourly weather is used as some sort of average, I think it's better to address then.